runtime/altair/src/lib.rs

```diff
@@ -1447,6 +1527,8 @@ construct_runtime!(
 		InterestAccrual: pallet_interest_accrual::{Pallet, Storage, Event<T>, Config<T>} = 101,
 		Investments: pallet_investments::{Pallet, Call, Storage, Event<T>} = 102,
 		PoolRegistry: pallet_pool_registry::{Pallet, Call, Storage, Event<T>} = 103,
+		BlockRewardsBase: pallet_rewards::<Instance1>::{Pallet, Storage, Event<T>, Config<T>} = 104,
```

Comment:
Not sure if we want to use `Instance2` here to be according to the development Instances, if not, later, it can be a mess. WDYT?

Reply:
Ok, I see your point.

> So I would rather propose to change it in the dev runtime instead.

Would it need a migration?

Reply:
It would not! During the offsite, we decided there should not be migrations for `dev`.

Reply:
It doesn't make sense to write migrations for a runtime which is only supposed to be used in ephemeral networks 🙃 If Demo was still based on `dev`, we would still be in that dilemma. That was the main motivation to base Demo on `altair` runtime.

Reply:
Ok, I understand 👍🏻. Agree totally with it